CMakeLists.txt

```diff
@@ -599,6 +599,15 @@ CONFIGURE_FILE(
 	IMMEDIATE @ONLY
 )
 
+# Tests (need init'd before subdirectories)
+FIND_PACKAGE(GTest)
+
+IF (GTest_FOUND)
```

Comment:
It'd be worth noting somewhere that people can pass CMAKE_DISABLE_FIND_PACKAGE_GTest=1 to avoid the tests even if they have gtest installed. Even better if you add an option like USE_SDLMIXER.

Reply:
I've added `USE_TESTS` with then-required GTest package.